tests/unit/test_table.py

```diff
@@ -0,0 +1,69 @@
+"""Tests for high-level Flow of OpenFlow 1.3."""
+from unittest import TestCase
```

Comment:
`TestCase` and `nose` related unit tests are no longer accepted on our code base to avoid tech debt, please refactor this test suite to be `pytest` oriented.

Reply:
Ok